.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,6 @@
 
 * [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.
+* [ ] Do not squash commits when merging this PR.
```

Comment:
I'm not a fan of negatives (to avoid double negatives) so how about phrasing it in the positive and checked by default?
```suggestion
* [x] Squash commits when if needed merging this PR.
```

Reply:
I like that.